src/main/java/com/opentable/db/postgres/embedded/EmbeddedPostgres.java

```diff
@@ -428,8 +433,8 @@ public static EmbeddedPostgres start() throws IOException
         private int builderPort = 0;
         private PgBinaryResolver pgBinaryResolver = new BundledPostgresBinaryResolver();
 
-        private ProcessBuilder.Redirect errRedirector = ProcessBuilder.Redirect.INHERIT;
-        private ProcessBuilder.Redirect outRedirector = ProcessBuilder.Redirect.INHERIT;
+        private ProcessBuilder.Redirect errRedirector = ProcessBuilder.Redirect.PIPE;
```

Comment:
?

Reply:
Can you elaborate on your question please?

Reply:
We chatted in-person.  We just clarified how the two redirection modes work generally.